integration/airflow/tests/extractors/test_extractors.py

```diff
@@ -61,3 +63,29 @@ def test_adding_extractors():
     count = len(extractors.extractors)
     extractors.add_extractor("test", PostgresExtractor)
     assert len(extractors.extractors) == count + 1
+
+
+@patch('airflow.hooks.base.BaseHook')
+def test_instantiate_abstract_extractors(mock_hook):
+    class SQLCheckOperator:
+        conn_id = "postgres_default"
+    
+    mock_hook.get_connection_from_secrets.return_value = Connection(
+        conn_id="postgres_default",
+        conn_type="postgres"
+    )
+    extractors = Extractors()
+    sql_check_operator = SQLCheckOperator()
+    extractors.instantiate_abstract_extractors(task=sql_check_operator)
+    sql_check_extractor = extractors.extractors["SQLCheckOperator"]
+    assert sql_check_extractor._get_scheme() == "postgres"
+
+
+@patch('airflow.models.connection.Connection')
+def test_instantiate_abstract_extractors_value_error(mock_conn):
+    class SQLCheckOperator:
+        conn_id = "notimplementeddb"
+    
+    with pytest.raises(ValueError):
+        extractors = Extractors()
+        extractors.instantiate_abstract_extractors(task=SQLCheckOperator())
```

Comment:
@mobuchowski @collado-mike these tests currently aren't working, I'm having some real trouble understanding how to mock the objects properly. Would appreciate some help here!